app/components/markdown/markdown.js

```diff
@@ -90,6 +92,14 @@ export default class Markdown extends PureComponent {
         return !scheme || this.props.autolinkedUrlSchemes.indexOf(scheme) !== -1;
     };
 
+    getMentionKeys = () => {
+        const mentionKeys = this.props.mentionKeys;
+        if (this.props.disableAtChannelMentionHighlight) {
+            return mentionKeys.filter((mention) => !(mention.key === '@all' || mention.key === '@channel' || mention.key === '@here'));
```

Comment:
I was curious what you think about the following approach. Rather than compare key with the text, we can do something with an array (which is the same thing but I feel maybe a little bit more intuitive) Thoughts? 
```suggestion
            const disabledChannelKeys = [ '@all', '@channel', '@here'];
            return mentionKeys.filter((mention) => !disabledChannelKeys.includes(mention.key));
```

Reply:
I like it - although I just did `['@all', '@channel', '@here'].includes` instead of defining a const above